integrationExamples/gpt/idward_segments_example.html

```diff
@@ -0,0 +1,112 @@
+<html>
+<head>
+    <script async src="../../build/dev/prebid.js"></script>
+    <script async src="https://www.googletagservices.com/tag/js/gpt.js"></script>
```

Comment:
this is out of date

Reply:
used `securepubads.g.doubleclick.net/tag/js/gpt.js` instead